cmd/queue/main_test.go

```diff
@@ -443,6 +443,38 @@ func TestQueueTraceSpans(t *testing.T) {
 	}
 }
 
+func BenchmarkProxyHandler(b *testing.B) {
+	var httpHandler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {}
+	server := httptest.NewServer(httpHandler)
+	serverURL, _ := url.Parse(server.URL)
```

Comment:
what if there is any error while parsing?

Reply:
Strictly theoretically sure, but here we know _the one exact URL_ we're passing.